apps/frontend/src/pages/root/components/circular-progress/circular-progress.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+const ZERO = 0;
+const HUNDRED = 100;
+const TWO = 2;
+const SIXTY = 60;
+
+const CircularProgress: React.FC = () => {
+	const [animatedValue, setAnimatedValue] = useState<number>(ZERO);
+	const radius = 43;
+	const circumference = TWO * Math.PI * radius;
+	const strokeDashoffset =
+		circumference - (animatedValue / HUNDRED) * circumference;
+
+	useEffect(() => {
+		const timer = setTimeout(() => {
+			setAnimatedValue(SIXTY);
+		}, HUNDRED);
+
+		return (): void => {
+			clearTimeout(timer);
+		};
+	}, []);
+
+	return (
+		<div className={styles["container"]}>
+			<svg
```

Comment:
Can we use Chart js? Since we already have it in our app. Only if it will not take a long time

Reply:
Hi @fshabanov , for the custom svg i think it is not available with chart js, but available with a different package react-chartsjs-2, the length of the code would be pretty much the same if this is implemented since I added animation so that it syncs with the balance wheel on refresh and on load. Let me know your thoughts about this one.

Reply:
No need to install another library, keep the current implementation